app/entry.client.tsx

```diff
@@ -16,23 +15,10 @@ async function hydrate() {
 	await i18next
 		.use(initReactI18next) // Tell i18next to use the react-i18next plugin
 		.use(LanguageDetector) // Setup a client-side language detector
-		.use(HTTPBackend) // Setup your backend
 		.init({
 			...i18n, // spread the configuration
 			// This function detects the namespaces your routes rendered while SSR use
 			ns: getInitialNamespaces(),
-			backend: {
-				loadPath: '/locales/{{lng}}/{{ns}}.json',
```

Comment:
I think it can be nice to keep an example on how to configure the loadPath. lots of companies prefer to use a folder with all the translations in one place like

```
/locales/en.json
/locales/fr.json
/locales/es.json
```

In 15 years I never worked in a company where we use multiple translations files for the same local on the same product

I'm not a fan of the `/en/common.json` convention because if you send the file to someone or a server, and you send it as `common.json` without the folder, the receiver can forget what you sent